From 9efb01d32f139827665def7c9f86d0f85b7864a1 Mon Sep 17 00:00:00 2001
From: P. Christeas <p_christ@hol.gr>
Date: Thu, 28 Apr 2011 13:01:00 +0300
Subject: [PATCH 1000/1001] base_report_designer: remove empty DTD file

After discussing with HMO, this file seems to be redundant, and can be
removed.

diff --git a/base_report_designer/openerp_sxw2rml/office.dtd b/base_report_designer/openerp_sxw2rml/office.dtd
deleted file mode 100644
index e69de29..0000000
-- 
1.7.4.4

